testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java

```diff
@@ -11,6 +9,10 @@
 
    public static final String GZIP_CONTAINER_QUALIFIER = "jbossas-manual-gzip";
 
+   public static final String ENCODING_CONTAINER_QUALIFIER = "jbossas-managed-encoding";
+
    public static final int GZIP_CONTAINER_PORT_OFFSET = 1000;
 
+   public static final int ENCODING_CONTAINER_PORT_OFFSET = 2000;

```

Comment:
FYI: Offset 2000 is ok for 3.6.1.SPx branch, but this could cause some cherry-pick conflicts in master, because master currently uses 6 additional containers: https://github.com/resteasy/Resteasy/blob/master/testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java#L32

Reply:
OK. When I bring this to master I'll offset by 7000, thanks for the alert.

Reply:
How about using 7000 here too?

Reply:
At the end of the day, this is not making much difference. I would go with the same on all branches, but I'm fine with the other solution too.